Formula/sqldiff.rb

```diff
@@ -0,0 +1,21 @@
+class Sqldiff < Formula
+  desc "Displays the differences between SQLite databases."
+  homepage "https://www.sqlite.org/sqldiff.html"
+  url "https://www.sqlite.org/2016/sqlite-src-3140200.zip"
+  version "3.14.2"
+  sha256 "52507e20c2757b24b703b43ede77ce464c8106c1658a5b357974c435aa0677a6"
+
+  def install
+    system "./configure", "--disable-debug", "--prefix=#{prefix}"
+    system "make", "sqldiff"
+    bin.install "sqldiff"
```

Comment:
There isn't. This formula is basically [sqlite-analyzer.rb](https://github.com/Homebrew/homebrew-core/blob/master/Formula/sqlite-analyzer.rb) with some slight changes.

Both sqldiff and sqlite-analyzer are tools contained in the SQLite source tree. There is also dbhash, which could be useful. See the Tools section in the [SQLite documentation](https://sqlite.org/docs.html) for more information. It might make sense to make all three available in one formula.